configuration/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure-iot/configuration",
-  "version": "2.1.2",
+  "version": "2.2.0",
```

Comment:
nit: why minor version bump? 2.1.3 seems like the right version for this change

Reply:
I wanted to differentiate from licensing updates to function changes.